packages/react/src/components/TreeView/TreeView.mdx

```diff
@@ -12,6 +13,28 @@ import { ArgsTable } from '@storybook/blocks';
 
 ## Overview
 
+TreeView is a component that allows users to navigate through a hierarchy of items. It is used to display nested lists of items that can be expanded or collapsed to show and hide additional items.
+
+<Canvas>
+  <Story of={TreeViewStories.Default} />
+</Canvas>
+
+## With Controlled Expansion
+
+The `expanded` prop can be used to control the expansion state of the TreeView. The `isExpanded` prop can be used to determine if an individual tree node is expanded or collapsed.
+
+<Canvas>
+  <Story of={TreeViewStories.WithControlledExpansion} />
+</Canvas>
+
+## With Custom Icons
+
+The `renderIcon` prop can be used to customize the icons of each node in the TreeView.
+
+<Canvas>
+  <Story of={TreeViewStories.WithCustomIcons} />
```

Comment:
Heads up, in #16910 this api is changing. If that's merged first, this will need updated. If this PR is merged first @guidari this will be a new spot to fix in #16910

Reply:
I'll merge it and then fix it